.github/workflows/release_pr.yml

```
@@ -38,11 +38,11 @@ jobs:
 
     - name: Create Branch and Pull Request
       run: |
-        NEW_VERSION="${{ steps.bump_version.outputs.new_version }}"
+        NEW_VERSION="${{ steps.bump_version.outputs.NEW_VERSION }}"
```

Ah, since the key is `new_version`:
```suggestion
        NEW_VERSION="${{ steps.bump_version.outputs.new_version }}"
```